src/lexer.coffee

```diff
@@ -480,16 +480,16 @@ exports.Lexer = class Lexer
         @validateUnicodeCodePointEscapes body, {delimiter}
         @token 'REGEX', "/#{body}/#{flags}", {length: end, origin, data: {delimiter}}
       else
-        @token 'REGEX_START', '(',    {length: 0, origin}
-        @token 'IDENTIFIER', 'RegExp', length: 0
-        @token 'CALL_START', '(',      length: 0
+        @token 'REGEX_START', '(',    {length: 0, origin, generated: yes}
```

Comment:
Need to mark all these fake tokens from converting an interpolated regex into a nonexistent `RegExp()` call as generated